src/AreaChart/AreaSeries/AreaSeries.tsx

```diff
@@ -116,140 +118,143 @@ interface AreaSeriesState {
 const PADDING = 25;
 const HALF_PADDING = PADDING / 2;
 
-export class AreaSeries extends Component<AreaSeriesProps, AreaSeriesState> {
-  static defaultProps: Partial<AreaSeriesProps> = {
-    colorScheme: 'cybertron',
-    animated: true,
-    interpolation: 'linear',
-    type: 'standard',
-    line: <Line />,
-    area: <Area />,
-    markLine: <MarkLine />,
-    tooltip: <TooltipArea />,
-    symbols: <PointSeries />
+// export class AreaSeries extends Component<AreaSeriesProps, AreaSeriesState> {
+//   static defaultProps: Partial<AreaSeriesProps> = {
+//     colorScheme: 'cybertron',
+//     animated: true,
+//     interpolation: 'linear',
+//     type: 'standard',
+//     line: <Line />,
+//     area: <Area />,
+//     markLine: <MarkLine />,
+//     tooltip: <TooltipArea />,
+//     symbols: <PointSeries />
+//   };
+export const AreaSeries: FC<Partial<AreaSeriesProps>> = ({
+  colorScheme = 'cybertron',
+  animated = true,
+  interpolation = 'linear',
+  type = 'standard',
+  line = <Line />,
+  area = <Area />,
+  markLine = <MarkLine />,
+  tooltip = <TooltipArea />,
+  symbols = <PointSeries />,
+  data,
+  height,
+  id,
+  width,
+  isZoomed,
+  xScale,
+  yScale
+}) => {
+  const initialAreaState: AreaSeriesState = {
+    activePoint: undefined,
+    activeValues: undefined
   };
+  const [areaState, setAreaState] = React.useState<AreaSeriesState>(
```

Comment:
should be `useState` and destructure above.

Reply:
fixed